packages/components/base/src/virtualized-list/index.tsx

```
@@ -17,41 +16,65 @@ function generateStableKey(item: React.ReactNode, generateUniqueId: () => string
-export interface VirtualizedListITem {
+export interface VirtualizedListRef {
+  /**
+   * Scrolls the list to the item at the specified index.
+   *
+   * @param index - The zero-based index of the item to scroll into view.
+   * @param options - Optional configuration for scrolling behavior (e.g., `behavior`, `block`, `inline`).
+   */
+  scrollToIndex: (index: number, options?: ScrollIntoViewOptions) => void;
+}
+
+export interface VirtualizedListItem {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'VirtualizedListItem'. Consider to delete it or rename it.

'VirtualizedListItem' is defined here:


(lint/suspicious/noRedeclare)